vm-setup/roles/v1aX_integration_test/templates/Metal3MachineTemplate.yml

```diff
@@ -1,10 +1,10 @@
-apiVersion: infrastructure.cluster.x-k8s.io/{{CAPM3_API_VERSION}}
+apiVersion: infrastructure.cluster.x-k8s.io/{{CAPM3_VERSION}}
 kind: Metal3MachineTemplate
 metadata:
   name: {{M3MT_NAME}}
   namespace: metal3
   ownerReferences:
-  - apiVersion: cluster.x-k8s.io/{{CAPI_API_VERSION}}
+  - apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
This should be parametrised as well. 
We could run the same test with v1alpha4 and v1alpha5 as well.

Reply:
Yes I know, but we cannot use `CAPI_VERSION` yet. In the CI it is set to `v1alpha4` and it doesn't work. See https://github.com/metal3-io/metal3-dev-env/issues/688

Reply:
Ok let's wait for v1alpha4 to land. Then I'll use `CAPI_VERSION` here. :slightly_smiling_face: